src/Compiler/CompilerPasses.cpp

```diff
@@ -178,12 +185,9 @@ void addONNXToMLIRPasses(mlir::PassManager &pm, bool targetCPU,
         onnx_mlir::createInstrumentPass(instrumentOps, instrumentActions));
 }
 
-void addONNXToKrnlPasses(mlir::PassManager &pm, int optLevel, bool enableCSE,
-    std::string ONNXOpsStatFormat) {
-  if (enableCSE)
-    // Eliminate common sub-expressions before lowering to Krnl.
-    // TODO: enable this by default when we make sure it works flawlessly.
-    pm.addPass(mlir::createCSEPass());
```

Comment:
Looks like this was removed by accident since `enableCSE=true` means we call `pm.addPass(mlir::createCSEPass());`.